my-app/src/main/java/com/mycompany/app/findInverseSquareRoot.java

```diff
@@ -0,0 +1,26 @@
+package com.mycompany.app;
+
+public class findInverseSquareRoot {
+
+		public static float Q_rsqurt(float x) {
```

Comment:
That's not how to name a method in java.

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
Done